epub34/authoring/index.html

```diff
@@ -592,14 +571,14 @@ <h3>Terminology</h3>
 					</dt>
 					<dd>
 						<p>A resource that contains content or instructions that contribute to the logic and rendering
-							of an [=EPUB publication=]. In the absence of this resource, [=reading systems=] may not
-							render the EPUB publication as the [=EPUB creator=] intends. Examples of publication
-							resources include the [=package document=], [=EPUB content documents=], CSS Style Sheets,
-							audio, video, images, embedded fonts, and scripts.</p>
-						<p>EPUB creators must list publication resources in the package document [=EPUB manifest |
-							manifest=] and typically bundle them all in the [=EPUB container=] (the exception being they
-							may locate resources listed in <a href="#sec-resource-locations"></a> outside the EPUB
-							container).</p>
+							of an [=EPUB publication=]. In the absence of this resource, [=reading systems=] might not
+							render the EPUB publication as intended. Examples of publication resources include the
+							[=package document=], [=EPUB content documents=], CSS Style Sheets, audio, video, images,
+							embedded fonts, and scripts.</p>
+						<p>The package document [=EPUB manifest | manifest=] has to include a list of all publication
+							resources and the resources typically have to be bundled in the [=EPUB container=] (the
+							exception being that resources listed in <a href="#sec-resource-locations"></a> can be
+							located outside the EPUB container).</p>
```

Comment:
```suggestion
						<p>The package document [=EPUB manifest | manifest=] has to include a list of all publication
							resources that typically have to be bundled in the [=EPUB container=] (the
							exception being that resources listed in <a href="#sec-resource-locations"></a> can be
							located outside the EPUB container).</p>
```

Avoiding word repetition...